<ui version="4.0" >
 <class>PasswordDlg</class>
 <widget class="QDialog" name="PasswordDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>578</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="Label_Bookmark" >
         <property name="text" >
          <string>Last File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Button_Bookmarks" >
         <property name="popupMode" >
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="arrowType" >
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Enter a Password and/or choose a key file.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupframe" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Key</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="pw_entry" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>556</width>
           <height>116</height>
          </rect>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="Check_Password" >
            <property name="text" >
             <string>Password:</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLineEdit" name="Edit_Password" >
              <property name="font" >
               <font>
                <family>Serif</family>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="ButtonChangeEchoMode" />
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="Check_KeyFile" >
            <property name="text" >
             <string>Key File:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QComboBox" name="Combo_KeyFile" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_Browse" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="Button_GenKeyFile" >
            <property name="text" >
             <string>Generate Key File...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pw_repeat" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>556</width>
           <height>116</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Please repeat your password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Edit_PwRepeat" >
            <property name="font" >
             <font>
              <family>Serif</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="Button_Back" >
              <property name="text" >
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="Label_Unequal" >
              <property name="font" >
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>Passwords are not equal.</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>Button_Bookmarks</tabstop>
  <tabstop>Check_Password</tabstop>
  <tabstop>Edit_Password</tabstop>
  <tabstop>ButtonChangeEchoMode</tabstop>
  <tabstop>Check_KeyFile</tabstop>
  <tabstop>Combo_KeyFile</tabstop>
  <tabstop>Button_Browse</tabstop>
  <tabstop>Button_GenKeyFile</tabstop>
  <tabstop>Edit_PwRepeat</tabstop>
  <tabstop>Button_Back</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
